doc/guides/backporting-to-release-lines.md

```diff
@@ -0,0 +1,124 @@
+# How to Backport a Pull Request to a Release Line
+
+## Staging branches
+
+Each release line has a staging branch that the releaser will use as a scratch
+pad while preparing a release. The branch name is formatted as follows:
+`vN.x-staging` where `N` is the major release number.
+
+### Active staging branches
```

Comment:
Might be best to avoid embedding "current" versions in here as they won't stay current long and this doc would need to get updated constantly.